src/tpl/main-content.html

```diff
@@ -30,6 +30,13 @@ <h5>
                 all'interno dell'Amministrazione stessa.
               </small>
             </div>
+            <div class="form-group">
+              <input type="tel" class="form-control" id="telReferente" name="telReferente" required>
+              <label for="nomeReferente">Tel. Referente *</label>
```

Comment:
I'd write the extended version
```suggestion
              <label for="nomeReferente">Telefono Referente *</label>
```